pkg/api/convertertoapi.go

```diff
@@ -174,6 +174,10 @@ func convertV20160330AgentPoolProfile(v20160330 *v20160330.AgentPoolProfile, api
 	api.FQDN = v20160330.FQDN
 	api.OSType = OSType(v20160330.OSType)
 	api.Subnet = v20160330.GetSubnet()
+	api.Attributes = map[string]string{}
+	for k, v := range v20160330.Attributes {
```

Comment:
already backed out the vlabs change in master. will back out of this branch shortly